perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/bevpool_plugin.hpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef AUTOWARE__TENSORRT_BEVDET__BEVPOOL_PLUGIN_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__BEVPOOL_PLUGIN_HPP_
+
+#include <NvInfer.h>
+#include <unistd.h>
```

Comment:
Consider adding more specific includes instead of `<unistd.h>`.  Including only the necessary header files enhances readability and reduces potential conflicts. Specify which functions from `<unistd.h>` are used and include the appropriate header instead (e.g., `<sys/types.h>` or `<sys/stat.h>` if dealing with file system calls).